doc/developer_guide.md

```diff
@@ -0,0 +1,237 @@
+# Setting up a development environment
+
+This guide describes how to install and configure the development environment for contributing to the colorcet repository.
+
+If you have any problems with the steps here, please reach out in the `dev` channel on [Discord](https://discord.gg/rb6gPXbdAr) or on [Discourse](https://discourse.holoviz.org/).
+
+## TL;DR
+
+0. Open an [issue on Github](https://github.com/holoviz/colorcet/issues) if needed
+1. Fork and clone [colorcet's Github repository](https://github.com/holoviz/colorcet)
+2. Install [`pixi`](https://pixi.sh)
+3. Run `pixi run setup-dev` to create your development environment
+4. Run `pixi shell` to activate the created environment
```

Comment:
This is not a needed step.

Reply:
Step 4? In my experience, `pixi run setup-dev` does not activate the environment after creating it.

Reply:
You don't need to activate the environment to run the tasks.

Reply:
OK